scss/product/components/_card.scss

```diff
@@ -313,3 +313,73 @@
     }
   }
 }
+
+//card with logo
+.card_header{
+  display: flex;
+  align-items: center;
+  >div{
+    padding:0;
+  }
+  .card_header-logo{
+    margin-right: 10px;
+    .table-avatar-image{
+        width: 50px;
+    }
+  }
+  .card_header-title{
+    max-width: 50%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    .card_header-title--text{
+      font-size:16px;
+      color: $gray-700;
+      font-weight: 600;
+    }
+    .card_header-meta {
+    font-size:12px;
+    margin-top: 5px;
+    span{
+      +span{
+      position: relative;
+      padding-left: 10px;
+      margin-left: 5px;
+        &::before{
+        content: '';
+        position: absolute;
+        left: 0;
+        top: 3px;
+        bottom: 3px;
+        width: 1px;
+        background: $secondary;
+        }
+      }
+      }
+    }
+  }
+}
+.card-details{
+    border-top: 1px solid $secondary;
+    margin-top: 15px;
+    padding-top: 15px;
+    .lists{
+        display: flex;
+        flex-wrap: wrap;
+        .list{
+            font-size:12px;
+            display: flex;
+            align-items: center;
+            +.list{
+            margin-top: 15px;
+            }    
+            .icon img{
+                max-width: 100%;
```

Comment:
can you attach a screenshot of this card. I feel this card is specific to one use case, like card--> lists--> icons.

Reply:
Yes. It's a specific use case. 
![Screenshot from 2020-04-03 11-37-11](https://user-images.githubusercontent.com/34312966/78329529-8fbd0200-759f-11ea-84a5-04ce2fd08e21.png)